MonitorWidget/Info.plist

```diff
@@ -2,6 +2,16 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSLocationUsageDescription</key>
+	<string>Um die stets die nächste favorisierte Haltestelle anzuzeigen, brauchen wir Zugriff auf Standortdaten.</string>
+	<key>CFBundleIdentifier</key>
+	<string></string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>Um die stets die nächste favorisierte Haltestelle anzuzeigen, brauchen wir Zugriff auf Standortdaten.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>Um die stets die nächste favorisierte Haltestelle anzuzeigen, brauchen wir Zugriff auf Standortdaten.</string>
```

Comment:
There is a typo: "die" should be "dir".

It would be better you use the word "benötigen" and not "brauchen".